docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -791,168 +791,55 @@ This section presents the FreeRADIUS configuration steps. In some occasions, a R
 
 Option 1: Authentication against Active Directory (AD)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In the Administration interface under *Configuration -> Domains*.
 
-Samba / Kerberos / Winbind
-++++++++++++++++++++++++++
+NOTE: If you can't access this section and you have previously configured your server to bind to a domain externally to PacketFence, make sure you run `/usr/local/pf/addons/AD/migrate.pl`
 
-Install Samba. You can either use the sources or use the package for your OS. For RHEL/CentOS, do:
+Click *Add Domain* and fill in the informations about your domain.
 
-----
-yum install samba krb5-workstation
-----
+image::docs/images/domain-add.png[scaledwidth="100%",alt="Adding a new domain"]
 
-For Debian and Ubuntu, do:
+Where :
 
-----
-apt-get install samba winbind krb5-user
-----
+* *Identifier* is a unique identifier for your domain. It's purpose is only visual.
+* *Workgroup* is the workgroup of your domain.
+* *DNS name of the domain* is the FQDN of your domain. The one that suffixes your account names.
+* *This server's name* is the name that the server's account will have in your Active Directory.
+* *DNS server* is the IP address of the DNS server of this domain. Make sure that the server you put there has the proper DNS entries for this domain.
+* *Username* is the username that will be used for binding to the server. This account requires administrative privileges.
+* *Password* is the password for the username defined above. Note that this password will be in *clear text* in the PacketFence configuration. We suggest you create an additionnal account specificly for this task.
```

Comment:
It's because it's used for test join and test authentication.
Also if the server has to be re-joined it's needed